JSDemos/Demos/Scheduler/GroupOrientation/description.md

```diff
@@ -1 +1,5 @@
-This example shows the **workWeek** view with different group orientations. Select the *"Vertical Grouping"* tab to&nbsp;see the resource headers organized in&nbsp;a&nbsp;column and the *"Horizontal Grouping"*&nbsp;&mdash; arranged in&nbsp;a&nbsp;row.
\ No newline at end of file
+[Resources](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/) are categories used to classify appointments. To understand what resources are, consider the following example: in an educational center lectures are held in several rooms; in Scheduler terms, room is a resource kind, individual rooms are resource instances, and lectures are appointments that use these resource instances.
+
+The [Resources demo](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) describes how to assign resources. When assigned, they can also be used to group appointments. To implement this, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. The order of resource headers in the UI is the same as the order of the resource instances. Note that if the **resources** array has more than one element, grouping is nested. 
```

Comment:
```suggestion
To group appointments by resources, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. The order of resource headers in the UI is the same as the order of the resource instances. Note that if the **resources** array has more than one element, grouping is nested. 
```

Reply:
> The order of resource headers in the UI is the same as the order of the resource instances.

By "resource headers" you mean "group headers"? "the order of the resource instances" in which array?

> Note that if the **resources** array has more than one element, grouping is nested. 

Do you mean "if the **groups** array has more than one element"?